### DISCUSSION TITLE
Make WorkerIndexHtml settable?

### DISCUSSION THREAD

**Discussion by softhorizons** [LostBeard/SpawnDev.BlazorJS#36] [Ideas]
Nowhere near as interesting as the previous topic (SIMD), but ... in WebServiceService.cs line 134, WorkerIndexHtml is private for set, yet I can find no code that actually sets it.
Is there any reason this couldn't be publicly settable? In my case there's no index.html file nor default -- it's iframe.html. I need to set this to iframe.html in Program.cs when creating the service.

**Comment by LostBeard** [comment]
It will be publicly settable in the next update..

**Comment by LostBeard** [comment]
Version 2.3.0 is up with the requested change.
